js/propExtractorConfig.ts

```diff
@@ -1,5 +1,76 @@
 import { PropExtractorConfig } from './util/extractProps';
 
+const REACT_NATIVE_ELEMENTS_CONFIG: PropExtractorConfig = {
+  // Interactive components.
+  Avatar: {
+    include: [
+      'editButton.name',
+      'editButton.type',
+      'icon.name',
+      'icon.type',
+      'showEditButton',
+      'title',
+    ],
+    exclude: [],
+  },
+  Badge: {
+    include: ['status', 'value'],
+    exclude: [],
+  },
+  Button: {
```

Comment:
I think that the way you have this now, this `Button` entry will override the default RN `Button` entry.  What we really need to do is do a deep merge instead of the object splat, I think...

Reply:
Alternately, we could just include `title` in here, though I like being able to keep them separate...